components/automate-ui/src/app/components/create-object-modal/create-object-modal.component.ts

```diff
@@ -43,12 +41,16 @@ export class CreateObjectModalComponent implements OnInit, OnChanges {
   }
 
   ngOnChanges(changes: SimpleChanges): void {
-    // if a new list of projects to populate dropdown with is passed in we update the dropdown
-    const checked = false;
-    if (hasIn('assignableProjects.currentValue', changes)) {
+    // update project dropdown if list changes
+    if (changes.assignableProjects) {
       this.projects = {};
       changes.assignableProjects.currentValue.forEach((proj: Project) =>
-        this.projects[proj.id] = { ...proj, checked });
+        this.projects[proj.id] = { ...proj, checked: false });
+    }
+    // clear checked projects when opening
+    if (changes.visible && (changes.visible.currentValue as boolean)) {
+      Object.values(this.projects).forEach(p => p.checked = false);
+      this.projectsUpdatedEvent.emit();
```

Comment:
This is half of the fix: resetting all the projects to unchecked upon opening the modal.